cmd/skopeo/proxy.go

```diff
@@ -116,6 +118,23 @@ type request struct {
 	Args []any `json:"args"`
 }
 
+type proxyErrorCode string
+
+const (
+	// proxyErrPipe means we got EPIPE writing to a pipe owned by the client
+	proxyErrPipe proxyErrorCode = "EPIPE"
+	// proxyErrRetryable can be used by clients to automatically retry operations
+	proxyErrRetryable = "retryable"
+	// All other errors
+	proxyErrOther = "other"
+)
```

Comment:
```suggestion
const (
	// proxyErrPipe means we got EPIPE writing to a pipe owned by the client
	proxyErrPipe proxyErrorCode = "EPIPE"
	// proxyErrRetryable can be used by clients to automatically retry operations
	proxyErrRetryable proxyErrorCode = "retryable"
	// All other errors
	proxyErrOther proxyErrorCode = "other"
)
```

Reply:
Ah sorry I'd missed that. Thanks, squashed in!